install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1098,6 +1098,10 @@ certgen:
   annotations:
     job: {}
     cronJob: {}
+  # ref: ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
```

Comment:
```suggestion
  # Node selector for certgen pods
  # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
```

Reply:
Thanks for the catch!
I've made some more changes, to align with the other values.